src/main/java/com/softserve/academy/spaced/repetition/controller/CategoryController.java

```diff
@@ -68,14 +65,14 @@
         Page<CategoryTopDTO> sortedCategoriesDTOS = categoryService
                 .getSortedCategories(pageNumber, sortBy, ascending).map((category) -> {
                     Link selfLink = linkTo(methodOn(CategoryController.class)
-                            .getCategoryById(category.getId())).withRel("category");
+                            .getCategoryById(category.getId())).withRel("categories");
```

Comment:
Maybe `withSelfRel`, because this method return list of categories and each category has a link that based on method `getCategoryById`, that return category, means.